docs/examples/library_factories/sqlalchemy_factory/test_example_1.py

```diff
@@ -1,4 +1,9 @@
-from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column
+import pytest
```

Comment:
I think we can just configure pytest to not include these tests regardless of the version. I'd rather not include these in the examples, since it'll just be more noise for the users reading the example.

What do you think, @adhtruong?

Reply:
Good point on noise on examples. I would still prefer these to be run in some way to ensure correctness. Would passing some sort of custom pytest CLI option or omitting docs examples when not on SQL2 and pydantic 2 be an alright alternative?

Reply:
I think the docs examples are only tested under `pydantic` v2 and `sqlalchemy` v2 as per the current workflow. So, it seems like the current working is already the second option you mentioned.

Reply:
Missed that. I'll revert these changes entirely then.